### PR TITLE
[s] New blob overminds off the station Z level are moved to the station

### PR DESCRIPTION
**Pull request by ghost**
Same as #32173 but done better

no more reebee blobs destroying the ark

🆑 More Robust Than You
fix: New blob overminds off the station Z level are moved to the station
/🆑

### CODE REVIEW

File: code/modules/mob/transform_procs.dm [hunk 1]
```diff
@@ -461,7 +461,10 @@
 	qdel(src)
 
 /mob/proc/become_overmind(starting_points = 60)
-	var/mob/camera/blob/B = new /mob/camera/blob(loc, starting_points)
+	var/turf/T = get_turf(loc) //just to avoid messing up in lockers
```
Comment (KorPhaeron) [inline code/modules/mob/transform_procs.dm]: Just check the area to see if its blob safe to stop them spawning in space or the ops shuttle or wherever as well

Comment (MoreRobustThanYou) [reply code/modules/mob/transform_procs.dm]: I made it _also_ check to see if it's blob safe

**Comment by Xhuis** [comment]
Reebe blobs sound completely hilarious and I'm totally fine with keeping them in. Still, if it's a problem, eh.

**Comment by MoreRobustThanYou** [comment]
@Xhuis i was a reebee blob once. I spawned next to the ark, during grace period... nobody saw me and the round ended in 30 seconds. OOC was super salty

**Comment by Hatterhat** [comment]
the fucking blob spawned on lavaland you retards speedmerge this immediately

**Pull request merged**